com.unity.mobile.notifications/Runtime/Android/Plugins/mobilenotifications.androidlib/build.gradle

```diff
@@ -12,6 +12,10 @@ def getCompileSdk(unityLib) {
 }
 
 android {
+    if (project.android.hasProperty("namespace")) {
```

Comment:
Could you add a comment that this is for Unity 21.3, so we can clean this up when dropped.
Also mention package in the manifest, I think it's not required after namespace is here.